### PR TITLE
Refactor: similar files between portals

### PR DESCRIPTION
**Pull request by Tguntenaar**
# Refactor Events Committee

As part of [917](https://github.com/NEAR-DevHub/neardevhub-bos/issues/917)

These change can be merged. Even though there is still some code debt, we should merge these improvements since the priorities have shifted to other stuff again.  I’ll address the remaining items once the indexer replacement is complete.

I used this list to keep track of refactoring (The molecule folder and its dependencies. Indented components are dependencies)
- [ ] molecule compose
	- [ ] SimpleMDE (infra has rfp) (events contains REPL_EVENTS)
		- [x] MarkdownEditor (deleted in devhub since it was just a proxy of SimpleMDE) (deleted in events-committee bc unused)
	- [x] MarkdownViewer (events contains REPL_EVENTS)
- [x] Profile line ( events contains REPL_EVENT ) (Infra reffert al naar devhub)
	- [x] BagdesList ( bestaat niet voor events )
- [x] Molecule.DropDown ( Only difference was test id)
- [x] Profile card

### CODE REVIEW

**Comment by Tguntenaar** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Tguntenaar Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Weekly streak is on the road, smart strategy! Secure your streak with another PR!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&pr=NEAR-DevHub/neardevhub-bos/916&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Tguntenaar?type=bot&pr=NEAR-DevHub/neardevhub-bos/916&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Tguntenaar)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Tguntenaar)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by petersalomonsen** [comment]
@race-of-sloths score 5

**Review by petersalomonsen — APPROVED** [review]

**Pull request merged**
